src/nspawn/nspawn.c

```diff
@@ -195,6 +194,7 @@ static char **arg_parameters = NULL;
 static const char *arg_container_service_name = "systemd-nspawn";
 static bool arg_notify_ready = false;
 static bool arg_use_cgns = true;
+static unsigned long arg_share_ns = CLONE_NEWIPC|CLONE_NEWPID|CLONE_NEWUTS;
```

Comment:
hmm, now that i am thinking about this, one more suggestion: the variable name suggests the contents of the variable was about sharing, but actually the bits are about what shall be detached. I'd thus suggest to rename the variable to arg_clone_flags or so?